CHANGELOG.md

```diff
@@ -110,12 +110,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid: Samus Returns
 
+- Added: Multiworld and item tracker support for Luma3DS.
```

Comment:
Minor pick, should we change this to say "support for Console"? seems easier to understand.